### PR TITLE
Use order time when instead current time

### PR DESCRIPTION
**Pull request by Zelig880**

### CODE REVIEW

**Comment by ryanmitchell** [comment]
Looks good to me once you tidy up the StyleCI.
@sampoyigi any objections to this commit being merged?

**Comment by Zelig880** [comment]
@ryanmitchell @sampoyigi All fixed for you

File: models/Coupons_model.php [hunk 1]
```diff
@@ -182,28 +181,33 @@ public function minimumOrderTotal()
...
 
-    public function isExpired()
+    /**
+     * Check if a coupone is expired
+     *
+     * @param Carbon\Carbon $orderDateTime orderDateTime
+     *
+     * @return bool
+     */
+    public function isExpired($orderDateTime)
```
Comment (sampoyigi) [inline models/Coupons_model.php]: For backward compatibility, make the `$orderDateTime` optional with a default value of null so we can use still use the current time when nothing is passed.

Something like this 

```
public function isExpired($orderDateTime = null) {
   if (is_null($orderDateTime))
      $orderDateTime = Carbon::now();
   ....
}
```

**Review by sampoyigi — CHANGES_REQUESTED** [review]

**Comment by Zelig880** [comment]
All changed and tested.

**Review by sampoyigi — APPROVED** [review]

**Pull request merged**